modules/utxo-lib/src/bitgo/parseInput.ts

```diff
@@ -11,16 +11,57 @@ export function isPlaceholderSignature(v: number | Buffer): boolean {
   return v === 0;
 }
 
-export interface ParsedSignatureScript {
+/**
+ * @return Checks if P2TR script path's control block matches BitGo's need
+ */
+export function isValidControlBock(controlBlock: Buffer): boolean {
+  return Buffer.isBuffer(controlBlock) && 33 <= controlBlock.length && controlBlock.length % 32 === 1;
```

Comment:
```suggestion
  // The last stack element is called the control block c, and must have length 33 + 32m
  return 33 <= controlBlock.length && controlBlock.length % 32 === 1;
```

Reply:
Done. But keeping Buffer type check as extra security for this public function. JS